lib/elixir/lib/module.ex

```diff
@@ -1270,9 +1271,16 @@ defmodule Module do
     case :ets.lookup(set, {:def, {name, arity}}) do
       [{_key, kind, meta, _, _, _}] ->
         clauses =
-          if options[:nillify_clauses],
-            do: nil,
-            else: bag_lookup_element(bag, {:clauses, {name, arity}}, 2)
+          cond do
+            options[:skip_clauses] ->
+              []
+
+            options[:nillify_clauses] ->
+              nil
```

Comment:
I think we can already emit a deprecation warning because I truly believe this option is not used anywhere.

We may even just remove the option/branch altogether? The whole thing is an optimization to load less data, so it is not like something will go wrong if we return the clauses.

Reply:
No usage whatsoever on Github.
So, warning or removal?

Reply:
I think we can remove it without loss of behaviour, yes.

Reply:
Done. Please have a look again